### ISSUE TITLE
Brackets issues in eclipse editor

### ISSUE THREAD

**Issue by barmat80** [arcticicestudio/nord#149]
Hello,
I reported an issue in "nord-eclipse-syntax" project, but no answers, so I'm linking the iusse here:

https://github.com/arcticicestudio/nord-eclipse-syntax/issues/12

I would also know if there will be a theme for eclipse, not only for the syntax editor.

Thank you

**Comment by arcticicestudio** [comment]
Hi @barmat80, please don't crate duplicate issues in multiple repositories due "I want my answer" now reasons.
There are humans behind open source projects that spend their free time to provide solutions **for free* to others. To request an instant answer feels like a hit in the face to all their efforts.
The Nord project has grown quite heavy and could be a full time job so please keep in mind that there are 500+ issues & PRs next to yours.

Anyway, I'll close this issue as duplicate of arcticicestudio/nord-eclipse-syntax#12. Please subscribe to this issue to get notified when someone comments or a PR get linked to it.

---

Duplicates arcticicestudio/nord-eclipse-syntax#12

**Comment by barmat80** [comment]
Hello,
Your post is totally unacceptable and very offensive to me as a human (because I'm a human too, if you didn't know...), because you're assuming I'm 2 yo child that wants his chocolate here and now.
I know that free time is very important for anyone (I don't even have enough for myself) and I'm aware that this is a free and open source project and that there are many issues next to mine with high priority.

First of all, I have to tell that I tried to fix the issue by myself and search all the projects to find something that could help, but no luck and so I decided to post the issues in the eclipse-syntax project.
Then I realized that the the majority of issues were opened in the main project and so I thought that issues in the eclipse-syntax project wasn't the right place to post.
So, I never ever wanted an instant answer and I didn't create a duplicate issue because "I want my answer".
If not so, I wouldn't link the old post, don't you think?

Maybe I made a mistake, so I apologize, but there was no absolutely reason for a brutal answer like the one you wrote and I'm sorry that you didn't understand this simple thing.

By the way, 
Happy Holidays
